PokeAlarm/Alarms/__init__.py

```diff
@@ -27,4 +27,4 @@ def alarm_factory(mgr, settings, max_attempts, api_key):
         from PokeAlarm.Alarms.Twitter import TwitterAlarm
         return TwitterAlarm(mgr, settings)
     else:
-        raise ValueError("%s is not a valid alarm type!".format(kind))
+        raise ValueError(f"{kind} is not a valid alarm type!")
```

Comment:
f-strings weren't introduced until python 3.6 - so you need to move your min version up